documentation/asciidoc/computers/camera/rpicam_apps_post_processing_writing.adoc

```diff
@@ -18,8 +18,6 @@ All post-processing stages must implement the following member functions:
 
 In any stage implementation, call `RegisterStage` to register your stage with the system.
 
-Don't forget to add your stage to the post-processing folder's `CMakeLists.txt` too!
```

Comment:
I _think_ we can remove this, since it refers to post-processing stages written by third parties, but included in rpicam. But if anyone knows the `meson` equivalent of `CmakeLists` where we should add the post-processing stages, let me know. I'd like to add a reminder here.

Reply:
We can use ``meson.build`` in place of ``CMakeLists.txt``

Reply:
Done, thanks.